### ISSUE TITLE
Modules with no imports trigger a warning

### ISSUE THREAD

**Issue by adamdelman** [Intsights/flake8-intsights#1]
The following code triggers the following incorrect warning:  E302 expected 2 blank lines, found 0
```python
class Example(
    Exception,
):
    pass

```
